src/Xamarin.Android.Build.Tasks/Utilities/NativeAssemblyGenerator/X86NativeAssemblyGenerator.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Xamarin.Android.Tasks
+{
+	abstract class X86NativeAssemblyGenerator : NativeAssemblyGenerator
+	{
+		protected override string LineCommentStart => "#";
+
+        protected X86NativeAssemblyGenerator (StreamWriter output, string fileName)
+			: base (output, fileName)
+		{}
+
+		protected override void ConfigureTypeMappings (Dictionary<Type, NativeType?> mapping)
+		{
+			base.ConfigureTypeMappings (mapping);
+
+			ConfigureTypeMapping<short>	 (".short", size: 2, alignment: 2);
+			ConfigureTypeMapping<ushort> (".short", size: 2, alignment: 2);
```

Comment:
Tabs after non-whitespace characters redux: this diff looks nice if I copy the `.patch` into an editor with 4 space tabs.  It doesn't look as nice in the GitHub web UI, which uses 8 space tabs.

You should use *only spaces* on a line once a non-whitespace character has been used.